qcodes/data/__init__.py

```diff
@@ -0,0 +1,6 @@
+from qcodes.data.location import FormatLocation
+from qcodes.data.data_array import DataArray
+from qcodes.data.format import Formatter
+from qcodes.data.gnuplot_format import GNUPlotFormat
+from qcodes.data.hdf5_format import HDF5Format
+from qcodes.data.io import DiskIO
```

Comment:
shouldn't these be relative imports in order to make the "module" independent on where it itself is located?

Reply:
fixed